pkg/daemon/update.go

```diff
@@ -890,8 +909,9 @@ func verifyUserFields(pwdUser ign3types.PasswdUser) error {
 	if tempUser.Name == constants.CoreUserName && len(tempUser.SSHAuthorizedKeys) >= 1 {
```

Comment:
It looks to me that verfiyUserField may fail (as else condition will run) in a situation  when rendered config have user core && PasswordHash != nil && SSHAuthorizedKeys  == nil

Reply:
This section was written by someone else- how should i resolve this?

Reply:
As discussed offline, you are free to make changes anywhere in MCO repo :) Also, it is good to fix any known regression(as a result of the change introduced) before merging PR